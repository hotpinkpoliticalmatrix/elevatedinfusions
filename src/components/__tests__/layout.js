import React from "react"
import renderer from "react-test-renderer"
import Layout, { PureLayout } from "../../components/layout"
import { StaticQuery } from "gatsby"

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: `Elevated Infusions Bakery`,
        },
      },
    })
  )
})

describe("PureLayout", () =>
  it("renders correctly", () => {
    const data = {
      site: {
        siteMetadata: {
          title: "Elevated Infusions Bakery",
        },
      },
    }

    const tree = renderer.create(<PureLayout data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
