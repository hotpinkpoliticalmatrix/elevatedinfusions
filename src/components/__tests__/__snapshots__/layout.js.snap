// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary Header-appBar-1 Header-transparent-14 Header-fixed-3 mui-fixed MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular Header-container-4 MuiToolbar-gutters"
    >
      <div
        className="Header-flex-5"
      >
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text Header-title-6"
          href="/#"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Elevated Infusions
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </a>
      </div>
      <div
        className="PrivateHiddenCss-smDown-23"
      >
        <ul
          className="MuiList-root HeaderLinks-list-33 MuiList-padding"
        >
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Home
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#products"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Products
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#menu"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Menu
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#reviews"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Reviews
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-describedby={null}
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="https://www.facebook.com/Elevateyourinfusions"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              target="_blank"
              title="Follow us on facebook"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 512 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-describedby={null}
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="https://www.instagram.com/elevateyourinfusions"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              target="_blank"
              title="Follow us on instagram"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 448 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="PrivateHiddenCss-mdUp-25"
    />
  </header>,
  <main />,
  <footer
    className="Footer-footer-70"
  >
    <div
      className="Footer-container-73"
    >
      <div
        className="Footer-left-68"
      >
        <ul
          className="MuiList-root Footer-list-74 MuiList-padding"
        >
          <li
            className="MuiListItem-root Footer-inlineBlock-75 MuiListItem-gutters"
            disabled={false}
          >
            Elevated Infusions
          </li>
          <li
            className="MuiListItem-root Footer-inlineBlock-75 MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="Footer-block-67"
              href="https://www.instagram.com/elevateyourinfusions/?rel='noopener'"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                />
              </svg>
            </a>
          </li>
          <li
            className="MuiListItem-root Footer-inlineBlock-75 MuiListItem-gutters"
            disabled={false}
          />
        </ul>
      </div>
      <div
        className="Footer-right-69"
      >
        © 
        2020
        , made with
         
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root Footer-icon-76"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
          />
        </svg>
         by
         
        <a
          className="Footer-a-71"
          href="https://staceyeliuk.dev?rel='noopener'"
          target="_blank"
        >
          hotPinkPoliticalMatrix
        </a>
         
        .
      </div>
    </div>
  </footer>,
]
`;

exports[`PureLayout renders correctly 1`] = `
Array [
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary Header-appBar-1 Header-transparent-14 Header-fixed-3 mui-fixed MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular Header-container-4 MuiToolbar-gutters"
    >
      <div
        className="Header-flex-5"
      >
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text Header-title-6"
          href="/#"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Elevated Infusions
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </a>
      </div>
      <div
        className="PrivateHiddenCss-smDown-23"
      >
        <ul
          className="MuiList-root HeaderLinks-list-33 MuiList-padding"
        >
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Home
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#products"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Products
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#menu"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Menu
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="/#reviews"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                Reviews
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-describedby={null}
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="https://www.facebook.com/Elevateyourinfusions"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              target="_blank"
              title="Follow us on facebook"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 512 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
          <li
            className="MuiListItem-root HeaderLinks-listItem-34 MuiListItem-gutters"
            disabled={false}
          >
            <a
              aria-describedby={null}
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root MuiButton-text RegularButton-button-45 RegularButton-transparent-59 HeaderLinks-navLink-36"
              href="https://www.instagram.com/elevateyourinfusions"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              target="_blank"
              title="Follow us on instagram"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 448 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="PrivateHiddenCss-mdUp-25"
    />
  </header>,
  <main />,
  <footer
    className="Footer-footer-70"
  >
    <div
      className="Footer-container-73"
    >
      <div
        className="Footer-left-68"
      >
        <ul
          className="MuiList-root Footer-list-74 MuiList-padding"
        >
          <li
            className="MuiListItem-root Footer-inlineBlock-75 MuiListItem-gutters"
            disabled={false}
          >
            Elevated Infusions
          </li>
          <li
            className="MuiListItem-root Footer-inlineBlock-75 MuiListItem-gutters"
            disabled={false}
          >
            <a
              className="Footer-block-67"
              href="https://www.instagram.com/elevateyourinfusions/?rel='noopener'"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                />
              </svg>
            </a>
          </li>
          <li
            className="MuiListItem-root Footer-inlineBlock-75 MuiListItem-gutters"
            disabled={false}
          />
        </ul>
      </div>
      <div
        className="Footer-right-69"
      >
        © 
        2020
        , made with
         
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root Footer-icon-76"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
          />
        </svg>
         by
         
        <a
          className="Footer-a-71"
          href="https://staceyeliuk.dev?rel='noopener'"
          target="_blank"
        >
          hotPinkPoliticalMatrix
        </a>
         
        .
      </div>
    </div>
  </footer>,
]
`;
